README.md

```diff
@@ -232,6 +230,9 @@ The summary includes the following fields:
 * `physicalMeasurementsFinalizedSite`: indicates the site where physical measurements were finalized for the participant
 * `signUpTime`: the time at which the participant initially signed up for All Of Us
 * `hpoId`: HPO marked as `primary` for this participant, if any (just the resource id, like `PITT` — not a reference like `Organization/PITT`)
+* `awardee`: An awardee a participant is paired with or "unset" if none.
```

Comment:
Make consistent with participant documentation; and it should be 'UNSET' rather than 'unset' I think?